content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_nomenclature.rst

```diff
@@ -0,0 +1,406 @@
++----------------------------------------+-------------+------------------------------+--------------------+--------------------+
+| Packaging date (YYMMDD)                | Packaging   | (13)(\\d{6})                 | Date               | Pack date          |
+|                                        | Date        |                              |                    |                    |
++----------------------------------------+-------------+------------------------------+--------------------+--------------------+
+| Best before date (YYMMDD)              | Best before | (15)(\\d{6})                 | Date               | Best before date   |
+|                                        | Date        |                              |                    |                    |
++----------------------------------------+-------------+------------------------------+--------------------+--------------------+
+| Expiration date (YYMMDD)               | Expiration  | (17)(\\d{6})                 | Date               | Expiry date        |
+|                                        | Date        |                              |                    |                    |
++----------------------------------------+-------------+------------------------------+--------------------+--------------------+
+| Variable count of items                | Quantity    | (30)(\\d{0,8})               | Measure            | UoM: Units         |
++----------------------------------------+-------------+------------------------------+--------------------+--------------------+
+| Count of trade items                   | Quantity    | (37)(\\d{0,8})               | Measure            | Qty in units for   |
+|                                        |             |                              |                    | containers (AI 02) |
++----------------------------------------+-------------+------------------------------+--------------------+--------------------+
+| Net weight: kilograms (kg)             | Quantity    | (310[0-5])(\\d{6})           | Measure            | Qty in kg          |
++----------------------------------------+-------------+------------------------------+--------------------+--------------------+
+| Length in meters (m)                   | Quantity    | (311[0-5])(\\d{6})           | Measure            | Qty in m           |
++----------------------------------------+-------------+------------------------------+--------------------+--------------------+
+| Net volume: liters (L)                 | Quantity    | (315[0-5])(\\d{6})           | Measure            | Qty in L           |
++----------------------------------------+-------------+------------------------------+--------------------+--------------------+
+| Net volume: cubic meters (:math:`m^3`) | Quantity    | (316[0-5])(\\d{6})           | Measure            | Qty in :math:`m^3` |
```

Comment:
The exponents in LaTex are super cool! They're looking a little out of place though, since the `math` formatting changes the font. But I found this `superscript` formatting that I think works perfectly!
```
m\ :sup:`3`
```
Using the `math` formatting for just the exponent could also work, although the exponent gets written in the math font:
```
m\ :math:`^3`
```

Reply:
Yayyy!! this workss